css/general.css

```diff
@@ -50,4 +50,111 @@ body {
 
 .search-container ::placeholder{
     color:white;
+}
+
+
+
+@media only screen and (max-width: 860px){
+
+    .navbar {
+        padding: 0.5rem 0.5rem;
+        justify-content: space-evenly;
+    }
+
+    .navbar-logo{
+        font-size:80%;
+    }
+
+    .search-container{
+        height: 2rem;
+    }
+
+    .cta .btn{
+       padding: 0.25rem 0.5rem;
+       font-size: 80%;
+    }
+
+    .cta .badge{
+        font-size:60%;
+    }
+
+    .search-container{
+        width:10rem;
+    }
+
+}
+
+
+@media only screen and (max-width: 540px){
+
+    .navbar {
+        padding: 0.25rem 0.1rem;
+        justify-content: space-evenly;
+    }
+
+    .navbar-logo{
+        font-size:50%;
+    }
+
+    .search-container{
+        height: 1.5rem;
+        font-size: 50%;
+    }
+
+    .search-icon{
+        margin-left: 0.25rem;
+    }
+
+    .search-input{
+        margin-left:0.5rem;
+    }
+
+
+    .cta .badge{
+        padding: 0.25rem 0.75rem;
+    }
+
+    .search-container{
+        width:7rem;
+    }
+
+    .notification-badge{
+        padding:0.1rem 0.3rem;
+    }
+}
+
+@media only screen and (max-width: 390px){
+    .search-container{
+        width:3.5rem;
+    }
+
+    .search-input{
+        margin-left:0.25rem;
+    }
+
+    .navbar{
+        padding: 0.2rem 0.1rem;
```

Comment:
Try to apply padding and margin in multiples of 4px/0.25rem or whatever you prefer but try to keep it consistent.

Reply:
Yes Mahesh will keep in mind from next time, thankyou :)